src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1985,7 +1985,7 @@ void searchContextProviders
 * 1. Preconditions
 * 2. Get the complete list of entities from mongo
 */
```

Comment:
Given that return type has changed in this function (void -> int), the function preamble comment should be edit to explain what is returned.